packages/ui/hooks/useRewards.ts

```diff
@@ -32,7 +32,6 @@ export function useRewards({ poolId, chainId }: UseRewardsProps) {
               return [];
             }),
             sdk.getFlywheelMarketRewardsByPool(poolData.comptroller).catch((exception) => {
-              // TODO LogRocket, this should never happen.
```

Comment:
TODO means we should add logging to logrocket, now sentry here. So we know when stuff goes wrong.

Reply:
Fixed